satpy/readers/geocat.py

```diff
@@ -56,7 +56,30 @@
 
 
 class GEOCATFileHandler(NetCDF4FileHandler):
-    """GEOCAT netCDF4 file handler."""
+    """GEOCAT netCDF4 file handler.
+
+    **Loading data with decode_times=True**
+
+    By default, this reader will use xarray_kwargs={"engine": "netcdf4", "decode_times": False}.
+    to match behavior of xarray when the geocat reader was first written.  To use different options
+    use reader_kwargs when loading the Scene:
+
+        scene = satpy.Scene(filenames,
+                            reader='geocat',
+                            reader_kwargs={'decode_times': True})
```

Comment:
Correct me if I'm wrong but this isn't how this would work. The user would be doing `reader_kwargs={"xarray_kwargs": {"decode_times": True}}`.

Reply:
You are correct fixed in  510c24